src/PersonObjects/Sleeve/ui/StatsElement.tsx

```diff
@@ -137,21 +137,21 @@ export function EarningsElement(props: IProps): React.ReactElement {
       [`Dexterity Exp:`, `${formatExp(CYCLES_PER_SEC * rates.dexExp)} / sec`],
       [`Agility Exp:`, `${formatExp(CYCLES_PER_SEC * rates.agiExp)} / sec`],
       [`Charisma Exp:`, `${formatExp(CYCLES_PER_SEC * rates.chaExp)} / sec`],
-      [`Reputation:`, <ReputationRate reputation={CYCLES_PER_SEC * repGain} />],
+      [`Reputation:`, <ReputationRate key="reputation" reputation={CYCLES_PER_SEC * repGain} />],
```

Comment:
This key could be renamed to reputation-rate to match the one for sleeve company work. The money-rate keys already match between SleeveClassWork and SleeveCompanyWork